app/controllers/step_by_step_pages_controller.rb

```diff
@@ -81,6 +81,9 @@ def schedule
       @parser = DatetimeParser.new(date: date_params, time: params[:schedule][:time])
       scheduled_at = @parser.parse
       if @parser.issues.any?
+        @parser.issues.each do |issue|
+          @step_by_step_page.errors.add :base, "Enter a valid #{issue.keys.first.to_s.humanize.downcase}"
```

Comment:
Could `issue.values` be used here instead?

Reply:
🤦‍♂ 

Yes! Good spot. Much better.